src/components/flat-table/flat-table-row/flat-table-row.component.js

```diff
@@ -128,7 +167,75 @@ const FlatTableRow = React.forwardRef(
       setIsExpanded(expanded);
     }, [expanded]);
 
-    return (
+    const draggableComponent = (
+      <DrawerSidebarContext.Consumer>
```

Comment:
I don't think we need to duplicate all the `DrawerSidebarContext` and sub row stuff.

Let's change this to 

```js
const rowComponent = (
<StyledFlatTableRow>
// With all props etc
</StyledFlatTableRow>
);

const draggableComponent = (
<DraggableComponent id={id} moveItem={moveItem} findItem={findItem}>
 {rowComponent}
</DraggableComponent>
);

return (
<DrawerSidebarContext.Consumer>
  {({ isInSidebar }) => (
    <>
      {draggable ? draggableComponent : rowComponent}
      // sub row stuff
    </>
  )}
 </DrawerSidebarContext.Consumer>
);
```